doc/airline.txt

```diff
@@ -1039,6 +1039,12 @@ neomake <https://github.com/neomake/neomake>
 * neomake warning >
   let airline#extensions#neomake#warning_symbol = 'W:'
 <
+-------------------------------------                    *airline-localsearch*
+neomake <https://github.com/mox-mox/localsearch>
```

Comment:
neomake?

Reply:
Sorry, I missed that one. It should be localsearch. Should I make a new pull request, or can you edit the existing PR?
(I don't mind creating a new PR, but I imagine that it might be easier to just change the existing one.)